CMakeLists.txt

```diff
@@ -130,16 +130,18 @@ set(OPTIONAL_LIBRARIES_STATIC)
 ########################################################################
 # LIBZMQ dependency
 ########################################################################
-find_package(libzmq REQUIRED)
+if (NOT libzmq_FOUND)
```

Comment:
add it here or it will be lost on the next iteration https://github.com/zeromq/zproject/blob/master/zproject_cmake.gsl

Reply:
Roger, what is the best way to test a build of zproject against czmq to make sure the output is the same? The change is relatively simple in the gsl file and if you guys are open to a PR on that _without_ me locally testing I'd prefer it (for the sake of keeping this simple on my end).

Reply:
it's a simple enough change and there's a CI if it breaks something, so just send it

Reply:
OK, I'll create the PR in that project and link it here to close this one out, thanks.